packages/contracts-bedrock/test/opcm/DeployOPCM.t.sol

```diff
@@ -0,0 +1,274 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Test, stdStorage, StdStorage } from "forge-std/Test.sol";
+import { DeployUtils } from "scripts/libraries/DeployUtils.sol";
+import { DeployOPCM, DeployOPCMInput, DeployOPCMOutput } from "scripts/deploy/DeployOPCM.s.sol";
+import { OPContractsManager } from "src/L1/OPContractsManager.sol";
+import { ISuperchainConfig } from "src/L1/interfaces/ISuperchainConfig.sol";
+import { IProtocolVersions } from "src/L1/interfaces/IProtocolVersions.sol";
+
+contract DeployOPCMInput_Test is Test {
+    DeployOPCMInput dii;
+    string release = "1.0.0";
+
+    function setUp() public {
+        dii = new DeployOPCMInput();
+    }
+
+    function test_getters_whenNotSet_reverts() public {
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.superchainConfig();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.protocolVersions();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.l1ContractsRelease();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.addressManagerBlueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.proxyBlueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.proxyAdminBlueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.l1ChugSplashProxyBlueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.resolvedDelegateProxyBlueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.anchorStateRegistryBlueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.permissionedDisputeGame1Blueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.permissionedDisputeGame2Blueprint();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.l1ERC721BridgeImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.optimismPortalImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.systemConfigImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.optimismMintableERC20FactoryImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.l1CrossDomainMessengerImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.l1StandardBridgeImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.disputeGameFactoryImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.delayedWETHImpl();
+
+        vm.expectRevert("DeployOPCMInput: not set");
+        dii.mipsImpl();
+    }
+
+    // Below setter tests are split into two parts to avoid stack too deep errors
+
+    function test_set_succeeds_part1() public {
```

Comment:
[CI](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/73499/workflows/f38de147-7159-434f-9411-9fa324453595/jobs/3005831) failing on the naming of this function, think it needs to be:
```suggestion
    function test_set_part1_succeeds() public {
```